### PR TITLE
Re-arrange simulation information in the documentation

### PR DESCRIPTION
**Pull request by Rensvandeschoot**
This PR proposes to re-arrange some information about the simulation options, the code to obtain the models, and to rename the chapter with all model.

### CODE REVIEW

File: docs/source/project_create.rst [hunk 1]
```diff
@@ -183,6 +183,7 @@ It is possible to change the settings of the Active learning model. There are
 four ingredients that can be changed in the software: the type of classifier,
 the query strategy, balance strategy, and the feature extraction technique.
 
+
```
Comment (J535D165) [inline docs/source/project_create.rst:186]:
```suggestion
```

File: docs/source/simulation_cli.rst [hunk 2]
```diff
@@ -21,45 +21,269 @@ simulate. The result of the simulation is stored, after a succesful
...
+
+
+You can also use one of the :ref:`benchmark-datasets
+<data_labeled:fully labeled data>` (see `index.csv
+<https://github.com/asreview/systematic-review-datasets/blob/master/index.csv>`_
+for dataset IDs).
+
+.. code:: bash
+
+    asreview simulate benchmark: [dataset_id]
```
Comment (J535D165) [inline docs/source/simulation_cli.rst:64]:
```suggestion
    asreview simulate benchmark:[dataset_id]
```

File: docs/source/simulation_cli.rst [hunk 3]
```diff
@@ -21,45 +21,269 @@ simulate. The result of the simulation is stored, after a succesful
...
+
+.. option:: --init_seed INIT_SEED
+
+    Seed for setting the prior indices if the prior_idx option is not used. If the option
+    prior_idx is used with one or more index, this option is ignored.
+
+
+
+Simulation Set-up
+~~~~~~~~~~~~~~~~~
```
Comment (J535D165) [inline docs/source/simulation_cli.rst:146]:
```suggestion
Simulation Setup
~~~~~~~~~~~~~~~~
```

File: docs/source/simulation_cli.rst [hunk 4]
```diff
@@ -21,45 +21,269 @@ simulate. The result of the simulation is stored, after a succesful
...
+  Some classifiers (models) and feature extraction algorithms require additional dependecies. Use :code:`pip install asreview[all]` to install all additional dependencies at once.
+
+
+.. _feature-extraction-table:
+
+Feature Extraction
+~~~~~~~~~~~~~~~~~~
+
++----------------+-----------------------------------------------------------+-----------------------------------------------------------------------------+
+| Name           | Reference                                                 | Requires                                                                    |
```
Comment (J535D165) [inline docs/source/simulation_cli.rst:215]: I propose to remove all these tables. See #1204 1204

Comment (Rensvandeschoot) [reply docs/source/simulation_cli.rst:215]: ok!

File: docs/source/data_format.rst [hunk 5]
```diff
@@ -80,7 +80,7 @@ names are allowed, see the table. It can be used for:
 - **Exploration**: You can explore a completely labeled dataset in the Exploration
   Mode. The relevant/irrelevant label in the dataset will be displayed on each record.
   This option is useful for training purposes, presentations, and workshops.
-- **Simulation**: In the :doc:`ASReview command line interface for simulations<cli/>`,
+- **Simulation**: In the :doc:`ASReview command line interface for simulations<simulation_cli/>`,
```
Comment (J535D165) [inline docs/source/data_format.rst:83]:
```suggestion
- **Simulation**: In a :doc:`Simulation<simulation_overview/>`,
```

**Pull request merged**